src/main/java/TestFile.java

```diff
@@ -13,22 +13,22 @@ private static void incorrectAssignmentInIfCondition() {
     }
 
     private static void incorrectComparingToItself() {
-        int x = 3423421;
-        if (x == x) {
```

Comment:
[Checkstyle] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2.